src/bp/core/database/tables/server-wide/telemetry_payloads.ts

```diff
@@ -0,0 +1,19 @@
+import { Table } from 'core/database/interfaces'
+
+export class TelemetryPayloadTable extends Table {
```

Comment:
Can we change that to TelemetryTable?

Reply:
fixed in 8255733510f7133ec177ecda0b7126723477099c